std/meta.d

```diff
@@ -61,6 +61,52 @@ unittest
 }
 
 /**
+ * Normally an alias cannot be declared of - for example - literal
```

Comment:
This line break should be removed because it makes the document strange.
See http://dtest.thecybershadow.net/artifact/website-c8fc79c53f070289e4ff7d2fd4b327fcaaedc90b-7cc6f4bcae27d4c4d8898db06da3cfc7/web/phobos-prerelease/std_meta.html#.Alias